src/suricata.c

```diff
@@ -2453,6 +2453,13 @@ int SCFinalizeRunMode(void)
     SCInstance *suri = &suricata;
     switch (suri->run_mode) {
         case RUNMODE_UNKNOWN:
+
+            SCLogMessage(SC_LOG_WARNING, NULL, 0, NULL, "runmodes",
```

Comment:
We don't use `SCLogMessage` directly. It is used by other functions from the SCLog API that have a more appropriate logging level, depending on the usecase. Here,  `SCLogWarning` or even `SCLogError`would work better.